jenkins/automation/kubernetes/setup-cortx-cluster.groovy

```diff
@@ -114,6 +114,10 @@ pipeline {
                 MASTER_NODE=$(head -1 "$HOST_FILE" | awk -F[,] '{print $1}' | cut -d'=' -f2)
                 scp -q "$MASTER_NODE":/root/deploy-scripts/k8_cortx_cloud/solution.yaml $WORKSPACE/artifacts/
                 cp /var/tmp/cortx-cluster-status.txt $WORKSPACE/artifacts/
+
+                # Copy the latest CORTX support bundle log tar file
+                LOG_FILE=$(ssh -o 'StrictHostKeyChecking=no' $MASTER_NODE 'ls -t /root | head -1')
```

Comment:
will this logic to fetch log file work each time?

Reply:
Yes updated the logic a little to fetch only logs-cortx-cloud tar files. Since the tart files are created at same location every time, the above logic will fetch the latest tar file always. This can be further worked on.